packages/client/mutations/GitHubCreateIssueMutation.ts

```diff
@@ -0,0 +1,48 @@
+import graphql from 'babel-plugin-relay/macro'
+import {commitMutation} from 'react-relay'
+import {StandardMutation} from '../types/relayMutations'
+import {GitHubCreateIssueMutation as TGitHubCreateIssueMutation} from '../__generated__/GitHubCreateIssueMutation.graphql'
+
+graphql`
+  fragment GitHubCreateIssueMutation_meeting on GitHubCreateIssueSuccess {
+    meetingId
```

Comment:
what's this fragment accomplish? if i create a new github issue do my team members see it show up?

Reply:
This is a placeholder. As the results aren't rendering (an issue I can work on but assume is outside the scope here), the issue will be created in GH but the viewer & team members won't see anything in Parabol.

I thought it'd be worth keeping this fragment here as we'll need it once the results are rendering but I'd be happy to remove it too

Reply:
let's treat i just like the jira version: https://github.com/ParabolInc/parabol/blob/d5633f04cd1c8761232da7c132d5e63e7968efa9/packages/client/mutations/JiraCreateIssueMutation.ts#L68